api/test/trace/noop_test.cc

```diff
@@ -9,22 +9,24 @@
 using opentelemetry::trace::NoopTracer;
 using opentelemetry::trace::Tracer;
 
-TEST(NoopTest, UseNoopTracers)
+//  Note: This test is no longer valid as Span no longer has field of tracer. Whether
+//  removing it depends on the creator of this file.
+TEST(NoopTest, DISABLE_UseNoopTracers)
```

Comment:
I think this should be `DISABLED_UseNoopTracers` instead of  `DISABLE_UseNoopTracers`. Then this code can be uncommented, and the test won't run anymore (see [Google Test docs](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#temporarily-disabling-tests)).

Reply:
Updated in new push